### DISCUSSION TITLE
about newest commit

### DISCUSSION THREAD

**Discussion by J4k-The-Reaper** [UltimaHoarder/UltimaScraper#1308] [Q&A]
how do i update  "api_helper.py"  and "onlyfans.py or is it automatic? do i have to do anything or just update requirements? i'm new at this stuff. sorry if i seem like a total noob.

**Comment by UltimaHoarder** [comment]
Download latest commit and overwrite the old contents with new

**Reply by J4k-The-Reaper** [reply]
I use the Github Windows app. Does it automatically do it when it asks to pull latest commits?

**Comment by UltimaHoarder** [comment]
Yeee it should

**Reply by J4k-The-Reaper** [reply]
Well good. Then it's up to date. Off topic question....will you be fixing script so we can add the "{date}_" back to config? It really does help organize content better.

**Reply by UltimaHoarder** [reply]
Yee, hopefully in 24hrs

**Reply by mediaburnwayne** [reply]
Did this get applied in the creator fix, or are we still waiting

**Reply by J4k-The-Reaper** [reply]
I belive we are still waiting.
